runtime/doc/ui.txt

```diff
@@ -379,6 +379,8 @@ numerical highlight ids to the actual attributes.
 	enough to cover the remaining line, will be sent when the rest of the
 	line should be cleared.
 
+	`wrap` is a boolean indicating whether this line wraps to the next row.
```

Comment:
It's a little awkward to fully document all of the details of `wrap`, since the implementation of line wrapping relies on some details that aren't super specified in the protocol:
 - The wrap flag is only set on the last grid_line update for a given line
 - The implementation of wrapping in the TUI assumes that the grid_line events will arrive in row order, and that a grid_line event with the wrap flag set will be immediately followed by a grid_line that updates the first column of the next row.

lmk if you have any suggestions on how this might be specified more precisely.

Reply:
It's ok to brain-dump whatever you can for now. We can always tighten it up later.

Reply:
Done, thanks!